### PR TITLE
Feature/hook up front end and initial test

### PR DESCRIPTION
**Pull request by lighthurst**
Hi @briansai Please review. Thank you.

https://trello.com/c/q3KxEa2u/14-hook-up-front-end-to-server

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Reservation from './Reservations'
```
Comment (briansai) [inline client/src/components/App.jsx:3]: Don't forget semicolon.

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  }
+  fetch (roomId) {
+    axios.get(`/rooms/${roomId}`)
+      .then(db => this.setState({availNights: db.data}))
+      .catch(err => console.error(err));
+  }
+  render() {
+    return (
+      <div>
+        <Reservation />
```
Comment (briansai) [inline client/src/components/App.jsx:24]: Will the availNights state be passed into Reservation as a prop?

File: client/src/components/Reservations.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+const Div = styled.div`
+  margin: 40px;
+  border: 5px outset pink;
+  &:hover {
+   background-color: yellow;
+ }
+`;
+
+const Reservation = () => (
```
Comment (briansai) [inline client/src/components/Reservations.jsx:12]: Assuming prop to be passed here from App.jsx?

**Review by briansai — APPROVED** [review]
Looks good.  Tried to follow through with the available nights information during your fetch, so I made some assumptions.

File: client/src/components/App.jsx (hunk 2 above)
Comment (lighthurst) [reply client/src/components/App.jsx:24]: Yes, at some point. Haven’t gotten there yet. Just checking to see if the data was getting sent to the client.

File: client/src/components/App.jsx (hunk 1 above)
Comment (lighthurst) [reply client/src/components/App.jsx:3]: Saweet, thank you.

**Pull request merged**